spec/features/items/index_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe "As a visitor, " do
```

Comment:
Since the user story mentions it's for all kinds of users/visitors, maybe we should change the describe block to something like "items index page" and add context blocks for different user types.